Resources/Prototypes/Catalog/Fills/Crates/bluespace_harvester.yml

```diff
@@ -0,0 +1,347 @@
+# Industrial Loot
+- type: entity
+  id: CrateHarvesterSingularity
+  parent: CrateLimitedSecure
+  description: You've got a bad feeling about this.
+  suffix: Bluespace Harvester, Singularity, DO NOT MAP
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: Singularity
+
+- type: entity
+  id: CrateHarvesterSingularityFake
+  parent: CrateLimitedSecure
+  description: You've got a bad feeling about this.
+  suffix: Bluespace Harvester, Singularity Fake
+  components:
+  - type: BluespaceHarvesterBundle
+    contents:
+    - id: SingularityToy
+      amount: 3
```

Comment:
I'm not sure, but a large number of objects with a SingularityDistortion component can hit client performance hard.

Reply:
The harvester should usually work at level 10, which is 32 points per second.  What will give out 1 industry box in 7 minutes, where with a 5% chance 3 toys drop out.  And calculating the odds, we get 3 toys every 2 hours and 20 minutes.